packages/faceted-search/src/components/Badges/BadgeTags/BadgeTags.scss

```diff
@@ -0,0 +1,49 @@
+.fs-badge-tags-form {
+	:global(.tc-tooltip-body) {
+		margin: 1.2rem 0 0;
+		padding: 0.5rem 2rem 2rem;
+		overflow-y: auto;
+		background: #fcfcfc;
```

Comment:
Isn't there a color in the color palette that suit the need instead of setting a new value ?

Reply:
No i try all colors available but nothing match well with it